From 9393816b39a898443099428a754edcc0b48cdaee Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Tue, 3 Oct 2017 15:11:01 +1000
Subject: [PATCH 4016/5855] amdgpu/dc: set a bunch of functions to static.

All of these are unused outside the file they are in.

Signed-off-by: Dave Airlie <airlied@redhat.com>
---
 drivers/gpu/drm/amd/display/dc/bios/bios_parser.c       | 2 +-
 drivers/gpu/drm/amd/display/dc/calcs/dcn_calcs.c        | 2 +-
 drivers/gpu/drm/amd/display/dc/core/dc.c                | 2 +-
 drivers/gpu/drm/amd/display/dc/core/dc_link.c           | 2 +-
 drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c        | 2 +-
 drivers/gpu/drm/amd/display/dc/core/dc_link_hwss.c      | 2 +-
 drivers/gpu/drm/amd/display/dc/dce/dce_audio.c          | 3 +--
 drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c | 6 +++---
 8 files changed, 10 insertions(+), 11 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c b/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c
index 6d2f886..aaaebd0 100644
--- a/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c
+++ b/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c
@@ -531,7 +531,7 @@ static enum bp_result bios_parser_get_hpd_info(struct dc_bios *dcb,
 	return BP_RESULT_NORECORD;
 }
 
-enum bp_result bios_parser_get_device_tag_record(
+static enum bp_result bios_parser_get_device_tag_record(
 	struct bios_parser *bp,
 	ATOM_OBJECT *object,
 	ATOM_CONNECTOR_DEVICE_TAG_RECORD **record)
diff --git a/drivers/gpu/drm/amd/display/dc/calcs/dcn_calcs.c b/drivers/gpu/drm/amd/display/dc/calcs/dcn_calcs.c
index 91f43a1..8ca6c3e 100644
--- a/drivers/gpu/drm/amd/display/dc/calcs/dcn_calcs.c
+++ b/drivers/gpu/drm/amd/display/dc/calcs/dcn_calcs.c
@@ -1135,7 +1135,7 @@ bool dcn_validate_bandwidth(
 		return false;
 }
 
-unsigned int dcn_find_normalized_clock_vdd_Level(
+static unsigned int dcn_find_normalized_clock_vdd_Level(
 	const struct dc *dc,
 	enum dm_pp_clock_type clocks_type,
 	int clocks_in_khz)
diff --git a/drivers/gpu/drm/amd/display/dc/core/dc.c b/drivers/gpu/drm/amd/display/dc/core/dc.c
index c8ccca9..242876d 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc.c
@@ -327,7 +327,7 @@ static void set_test_pattern(
 			cust_pattern_size);
 }
 
-void set_dither_option(struct dc_stream_state *stream,
+static void set_dither_option(struct dc_stream_state *stream,
 		enum dc_dither_option option)
 {
 	struct bit_depth_reduction_params params;
diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_link.c b/drivers/gpu/drm/amd/display/dc/core/dc_link.c
index 85742a4..9508072 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_link.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_link.c
@@ -208,7 +208,7 @@ static bool detect_sink(struct dc_link *link, enum dc_connection_type *type)
 	return false;
 }
 
-enum ddc_transaction_type get_ddc_transaction_type(
+static enum ddc_transaction_type get_ddc_transaction_type(
 		enum signal_type sink_signal)
 {
 	enum ddc_transaction_type transaction_type = DDC_TRANSACTION_TYPE_NONE;
diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c b/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c
index 9f00ffd..85d9738 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c
@@ -1209,7 +1209,7 @@ static inline bool reached_minimum_link_rate(enum dc_link_rate link_rate)
 	return link_rate <= LINK_RATE_LOW;
 }
 
-enum dc_lane_count reduce_lane_count(enum dc_lane_count lane_count)
+static enum dc_lane_count reduce_lane_count(enum dc_lane_count lane_count)
 {
 	switch (lane_count) {
 	case LANE_COUNT_FOUR:
diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_link_hwss.c b/drivers/gpu/drm/amd/display/dc/core/dc_link_hwss.c
index b30319f..5f815ca 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_link_hwss.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_link_hwss.c
@@ -110,7 +110,7 @@ void dp_enable_link_phy(
 	dp_receiver_power_ctrl(link, true);
 }
 
-bool edp_receiver_ready_T9(struct dc_link *link)
+static bool edp_receiver_ready_T9(struct dc_link *link)
 {
 	unsigned int tries = 0;
 	unsigned char sinkstatus = 0;
diff --git a/drivers/gpu/drm/amd/display/dc/dce/dce_audio.c b/drivers/gpu/drm/amd/display/dc/dce/dce_audio.c
index 6e94028..526ec5c 100644
--- a/drivers/gpu/drm/amd/display/dc/dce/dce_audio.c
+++ b/drivers/gpu/drm/amd/display/dc/dce/dce_audio.c
@@ -833,8 +833,7 @@ void dce_aud_wall_dto_setup(
 	}
 }
 
-bool dce_aud_endpoint_valid(
-		struct audio *audio)
+static bool dce_aud_endpoint_valid(struct audio *audio)
 {
 	uint32_t value;
 	uint32_t port_connectivity;
diff --git a/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c b/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
index a09727f..4fd49a1 100644
--- a/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
+++ b/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
@@ -1183,7 +1183,7 @@ static const struct audio_clock_info audio_clock_info_table_48bpc[14] = {
 
 };
 
-union audio_cea_channels speakers_to_channels(
+static union audio_cea_channels speakers_to_channels(
 	struct audio_speaker_flags speaker_flags)
 {
 	union audio_cea_channels cea_channels = {0};
@@ -1217,7 +1217,7 @@ union audio_cea_channels speakers_to_channels(
 	return cea_channels;
 }
 
-uint32_t calc_max_audio_packets_per_line(
+static uint32_t calc_max_audio_packets_per_line(
 	const struct audio_crtc_info *crtc_info)
 {
 	uint32_t max_packets_per_line;
@@ -1238,7 +1238,7 @@ uint32_t calc_max_audio_packets_per_line(
 	return max_packets_per_line;
 }
 
-void get_audio_clock_info(
+static void get_audio_clock_info(
 	enum dc_color_depth color_depth,
 	uint32_t crtc_pixel_clock_in_khz,
 	uint32_t actual_pixel_clock_in_khz,
-- 
2.7.4

